doc/man3/EC_KEY_new.pod

```diff
@@ -102,7 +102,9 @@ that it is valid.
 The functions EC_KEY_get0_group(), EC_KEY_set_group(),
 EC_KEY_get0_private_key(), EC_KEY_set_private_key(), EC_KEY_get0_public_key(),
 and EC_KEY_set_public_key() get and set the EC_GROUP object, the private key,
-and the EC_POINT public key for the B<key> respectively.
+and the EC_POINT public key for the B<key> respectively. The function
+EC_KEY_set_private_key() accepts NULL as the priv_key argument to securely clear
```

Comment:
In the SYNOPSIS section the argument is called `prv`, we should use the same name in all sections.

Personally, I am OK with editing the line in SYNOPSIS to call it `priv_key` instead, if you believe it improves readability.
Alternatively DESCRIPTION and RETURN VALUES edits should use `prv`.

@t8m do you have a preference on either approach?

Reply:
Also @t8m might help me remembering the appropriate formatting "tag" for argument names and function names: on that regard it seems the whole page needs reformatting so maybe we can defer that to a separate dedicated reformatting PR?

Reply:
Just fixed the `priv_key` issue, thanks!